src/gatling/scala/SingleLeaseMultipleChecks.scala

```diff
@@ -11,18 +10,20 @@ class SingleLeaseMultipleChecks extends Simulation {
   val authenticator: GoogleAuthenticator = new GoogleAuthenticator()
 
   val requestJwt = exec(
-    http("Lease token")
-      .post("/lease")
-      .formParam("microservice", config.getString("service.name"))
-      .formParam("oneTimePassword", "${otp}")
-      .check(bodyString.saveAs("jwt"))
+      http("Lease token")
+        .post("lease")
+        .proxy(Proxy("proxyout.reform.hmcts.net", 8080))
```

Comment:
Do we need a proxy always ?

Reply:
Would be good to make it optional with environment variables.